cmd/norduser/main.go

```diff
@@ -61,20 +61,11 @@ func addAutostart() (string, error) {
 }
 
 func startTray(quitChan chan<- norduser.StopRequest) {
-	try := 0
-	// Retry checking systray availability, as it might not be availalble on startup.
 	for {
 		if systray.IsAvailable() {
 			break
 		}
-
-		if try == 5 {
```

Comment:
Atm this is best way to do this, since we do not know when systray might become available. We check every 10 sec(in another thread) until systray becomes available or norduser shuts down.